store/iavl/store.go

```diff
@@ -45,7 +47,11 @@ func LoadStore(db dbm.DB, id types.CommitID, lazyLoading bool, startVersion int6
 // to the one given. Internally, it will load the store's version (id) from the
 // provided DB. An error is returned if the version fails to load.
 func LoadStoreWithInitialVersion(db dbm.DB, id types.CommitID, lazyLoading bool, initialVersion uint64) (types.CommitKVStore, error) {
-	tree, err := iavl.NewMutableTreeWithOpts(db, defaultIAVLCacheSize, &iavl.Options{InitialVersion: initialVersion})
+	cacheSize := viper.GetUint64(iAVLCacheSizeFlag)
```

Comment:
move this segment to a singleton mode execution to avoid muitl times call with ` viper.GetUint64`
```go
cacheSize := viper.GetUint64(iAVLCacheSizeFlag)
if cacheSize == 0 || int(cacheSize) < 0 {
		cacheSize = defaultIAVLCacheSize
	}
```